app/assets/stylesheets/shared/bootstrap-extensions.css.scss

```diff
@@ -10,3 +10,35 @@
     }
   }
 }
+
+#chat-with-us {
+  margin-bottom: 40px;
+
+  &--heading {
```

Comment:
These styles are not applied, good catch :+1: 

@kraytruthg remove it? :blush: